editor/editor.cpp

```diff
@@ -36,7 +36,8 @@ int main() {
 #endif
 
     InputManager::addCallback(InputKeyButton{Key::ESCAPE, InputKeyEventType::PRESSED, []{
-        Engine::getWindow()->shouldStopAfterThisFrame();
+        if(auto window = dynamic_cast<Window*>(Engine::getWindow()))
```

Comment:
Should be a better way to do this than type checking, dynamic casts are very slow